lib/helpers/is-geojson-table.js

```diff
@@ -0,0 +1,15 @@
+const _ = require('lodash')
+const getGeometryTypeFromGeojson = require('./get-geometry-type-from-geojson')
+
+module.exports = function isTable (data = {}) {
+// geometry indicates this in not a table
+  const geometryType = getGeometryTypeFromGeojson(data)
+  if (geometryType) return false
+
+  // Check for a valid fullExtent. If unset, assume this is a Table
+  const fullExtent = data.fullExtent || (data.metadata && data.metadata.fullExtent)
+  if (_.isUndefined(fullExtent) || _.isUndefined(fullExtent.xmin) || _.isUndefined(fullExtent.ymin) || fullExtent.xmin === Infinity) return true
```

Comment:
Can you move all of this to a helper called `hasValidFullExtent`?  Then you could just do:

`return hasValidFullExtent(args)`

Reply:
This was existing code just moved from the utils class.

Reply:
It's a pretty small ask.